data/migrations/20200109191108_initial.js

```diff
@@ -0,0 +1,36 @@
+
+exports.up = async function(knex) {
+  await knex.schema.createTable('transmission', (table) => {
+    table.increments('id')
+    table.string('type')
+        .unique()
+        .notNullable();    
+  });
+  await knex.schema.createTable('title', (table) => {
+    table.increments('id')
+    table.string('status')        
+        .notNullable()
+            
+  });
```

Comment:
transmission and title have a 1:1 relationship with a car so they should just be columns on the car table.